### ISSUE TITLE
Empty "Thank you for joining us" page

### ISSUE THREAD

**Issue by RemcoSimonides** [blockframes/blockframes#5715]
When having two tabs open on landing page https://staging.archipelmarket.com/ and when you login on one of the two; the other one shows this page:
![image.png](https://images.zenhubusercontent.com/5ed89f5adc98f2d44c81b9c0/cdf48c6b-6d3a-4dec-9531-bd62cdef21da)

**Comment by Mussuranae** [comment]
I got this error also on local environment. I had only one tab opened. I was connected with this user `vYluCjIoB5P6wjgg54azmoBbeaG2`, but get some strange behavior on my work. I decided to clear the cookie and refresh the page to get the landing page. I connected with the same user and get this error with this message.
![image](https://user-images.githubusercontent.com/48033264/117271149-32f2e180-ae5a-11eb-92b0-40577af9cca9.png)

Hope it can help to debug this issue.
